src/style.css

```diff
@@ -0,0 +1,92 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir.ttc);
+  font-style: normal;
+  font-weight: 600;
+}
+
+html {
+  font-family: Avenir, Arial, Helvetica, sans-serif;
+}
+
+.search-bar {
+  box-sizing: border-box;
+  margin-top: 20px;
+}
+
+.input::placeholder {
+  font-size: 16px;
+  color: #3d4e61;
+}
+
+.input--small::placeholder {
+  font-size: 14px;
+  color: #3d4e61;
+}
+
+.input {
+  background-image: url("./images/Search.svg");
+  background-repeat: no-repeat;
+  background-color: #fff;
+
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+
+  width: 100%;
```

Comment:
Correct right [css order properties](https://9elements.com/css-rule-order/). Fix it everywhere